debian/libndn-cxx0.symbols

```diff
@@ -0,0 +1,8500 @@
+libndn-cxx.so.0.5.0 libndn1 #MINVER#
```

Comment:
This line is wrong. The first parameter should be the SONAME matching what you're patching. I.e., `libndn-cxx.so.0` with your current patch or `libndn-cxx.so.0.5` with my suggestion. The second parameter is the package name. Currently `libndn-cxx0` or `libndn-cxx0.5` with my suggestion.